docs/configuration/misc.rst

```diff
@@ -44,3 +44,54 @@ Interval & duration fields in job definitions, pruning configurations, etc. must
 
     var durationStringRegex *regexp.Regexp = regexp.MustCompile(`^\s*(\d+)\s*(s|m|h|d|w)\s*$`)
     // s = second, m = minute, h = hour, d = day, w = week (7 days)
+
+
+.. _conf-include-directories:
+
+Including Configuration Directories
+-----------------------------------
+
+It is possible to distribute zrepl job definitions over multiple YAML files. This is
+achieved by using the `include` key in the configuration file.
+
+The directive is mutually exclusive with any other jobs definition and can only exist
+in the configuration file:
+
+::
+   global: ...
+
+   jobs:
+     include: jobs.d
+
+.. NOTE::
+   The included directory path is treated as absolute when starting with `/` else it
+   is treated as a relative path from the directory of the loaded configuration file.
+
+Included YAML job files must end with the `.yml` extension and can only contain
+contain the `jobs` key. Additionally, job names must be unique across all job YAML
+files.
+
+Examples
+^^^^^^^^
+
+::
+  > /etc/zrepl/zrepl.yml
+
+   jobs:
+     include: jobs.d
```

Comment:
So, it's all or nothing? Either I specify a list of jobs in the main config file or I specify a path where the files are located?

Reply:
Also, on the relative paths, I suggest to use `./jobs.d` so that it's clear this is a relative path and not some string.

Reply:
Actually thinking more about this if we have the include as a separate top level key we don't need this mutual exclusivity.

Will update the PR.